app/assets/v2/js/activity.js

```diff
@@ -8,7 +8,7 @@ $(document).ready(function() {
 
   var linkify = function(new_text) {
     new_text = new_text.replace(/(?:^|\s)#([a-zA-Z\d-]+)/g, ' <a href="/?tab=search-$1">#$1</a>');
-    new_text = new_text.replace(/\B@([a-zA-Z0-9_-]*)/g, ' <a data-usercard="$1" href="/profile/$1">@$1</a>');
+    new_text = new_text.replace(/(^|\s)@([a-zA-Z0-9_-]*)/g, '$1<a data-usercard="$2" href="/profile/$2">@$2</a>');
```

Comment:
is this $1 right?

Reply:
I believe that $1 was fat finger https://regex101.com/r/5H3V5y/1 but tell me if Im wrong

Reply:
Yes it is. It copies over the first capturing group which checks for beginning or whitespace (`(^|\s)`)

Reply:
This is what you'd get without $1 or any space.

<img width="197" alt="Screenshot 2020-04-22 at 1 56 16 AM" src="https://user-images.githubusercontent.com/36922376/79910523-81defc00-843c-11ea-85ac-8fa4f2b5461e.png">

Reply:
ohh totally make sense, thanks for explaining